lib/crypto/secp256k1/secp256k1.go

```diff
@@ -102,6 +106,44 @@ func NewKeypairFromPrivate(priv *PrivateKey) (*Keypair, error) {
 	}, nil
 }
 
+// NewKeypairFromMnenomic returns a new Keypair using the given mnemonic and password.
+func NewKeypairFromMnenomic(mnemonic, password string) (*Keypair, error) {
+	seed, err := schnorrkel.SeedFromMnemonic(mnemonic, password)
+	if err != nil {
+		return nil, err
+	}
+	return NewKeypairFromSeed(seed[:32])
+}
+
+// NewKeypairFromSeed generates a new secp256k1 keypair from a 32 byte seed
+func NewKeypairFromSeed(seed []byte) (*Keypair, error) {
+	// reference:
```

Comment:
This comment is not needed since this is pretty naive function and referencing substrate code is not necessary

Reply:
agreed